### DISCUSSION TITLE
[Script request] Snipe-IT

### DISCUSSION THREAD

**Discussion by aldekmt** [community-scripts/ProxmoxVE#365] [Request script] [answered]
### Application Name

Snipe-IT

### Website

https://snipeitapp.com/

### Description

Snipe-IT is an open-source asset management application designed to help IT teams efficiently track hardware, licenses, and other assets. With its user-friendly interface and robust API, Snipe-IT provides a centralized solution for managing and documenting assets in IT environments.

Adding an automated installation script for Snipe-IT to the Proxmox VE Helper-Scripts repository would greatly simplify the deployment process, especially in virtualized setups.

Why should Snipe-IT be added?

- Relevance for Proxmox Users: Many Proxmox users manage IT infrastructure and could benefit from streamlined asset management.
- Efficiency: Automating the setup process reduces manual configuration time, minimizing errors and improving consistency.
- Community Value: Including Snipe-IT in the Helper-Scripts would enhance Proxmox's utility by supporting a critical IT management need.

This integration would be a valuable addition for IT administrators seeking seamless solutions for infrastructure management.

An Easy way for this would be a Game-Changer for me, and maybe i am not alone.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by fidst** [comment]
I wonder if this would have to be multiple scripts. Once for the snipeit app, and another for a database.

Possibly a vm with docker installed and it automates both docker containers.

**Reply by michelroegl-brunner** [reply]
There is already an existing Docker container, inlcuding the DB, in the Official SnipeIT Repo. 
I created a Pull Request for it: https://github.com/community-scripts/ProxmoxVE/pull/507

**Reply by michelroegl-brunner** [reply]
I have a Bare Metal script working: https://github.com/community-scripts/ProxmoxVE/pull/538

**Comment by newmy-de** [comment, marked as answer]
It’s available now.

https://community-scripts.github.io/ProxmoxVE/scripts?id=snipeit

**Reply by aldekmt** [reply]
A big thank you for implementing this! 🙌 There’s already a demo on the website, but with the help of these helper scripts, it’s now possible to self-host and securely input sensitive data. This is a huge step forward—much appreciated!
